spec.bs

```diff
@@ -588,6 +589,14 @@ members:
         credential(s). The caller does not need to specify the
         [[#sctn-payment-extension-registration| payment extension]]; it is added
         automatically.
+
+    :  <dfn>localeHint</dfn> member
+    :: An optional list of **TODO, ISO-something?** locale descriptors, in
+        decreasing order of priority, that act as a hint from the website as to
+        what locale the users experience on that website is currently in.
+
+        NOTE: **TODO** - add a note as to how this is distinct from the dir/lang
```

Comment:
Perhaps:

> The `locale` is distinct from language or direction metadata associated with specific input members in that it represents the caller's requested localized experience, rather than an assertion about a specific string value.

Reply:
Done, thanks!